### PR TITLE
Change mosaic size inputs to sliders

### LINKED ISSUE #38: Change dimension values to sliders
I was thinking it might make more sense visually to have the 'width' and 'height' parameters show up as sliders, instead of text boxes. 

Or, maybe even both, with the slider & text box changing each other. This might not be very difficult to implement, given v-model, [v-slider](https://vuetifyjs.com/en/components/sliders), and [v-text-field](https://vuetifyjs.com/en/components/text-fields). 

Input would be appreciated.

### PR DESCRIPTION
**Pull request by centipeda**
Closes #38.

This branch changes the mosaic size inputs, which are currently text fields, to sliders:

image

The sliders are implemented with Vuetify's `<v-slider>` element. They should move simultaneously when "Preserve aspect ratio" is enabled. 

With this pull request, I believe that all size input is now fully validated -- the sliders enforce a certain minimum and maximum value, and the text boxes will always enforce a minimum/maximum value (and refuse to set with an improper value, like a letter or a negative number). As such, this might also resolve (possibly) #44.

One potential bug with this is that the aspect ratio is not correctly preserved on a rectangular mosaic when one of the sliders is dragged all the way to 1 (the mosaic becomes square instead). 

This could possibly be fixed by making the aspect ratio a persistent property of the Settings component, instead of a value calculated every time the size is supposed to be changed. Or it could just be intended behavior. Let me know what you think.

### CODE REVIEW

**Comment by centipeda** [comment]
I also had to increase the width of the settings panel by a fair bit (10em) to comfortably accommodate the text box and the slider on the same line. Testing the new layout on my phone and in the Chrome Devtools, it doesn't seem to be an issue (it's smart enough to resize the slider to fit the layout) but it can be changed if it doesn't work for somebody.

File: src/components/Settings.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card max-width="20em" max-height="100%" style="overflow: auto;">
+  <v-card max-width="30em" max-height="100%" style="overflow: hidden;">
```
Comment (stephen-hannon) [inline src/components/Settings.vue]: Is there a reason you changed overflow to hidden? Now you can't scroll on short screens.

Comment (centipeda) [reply src/components/Settings.vue]: Oh, whoops. I thought it was changing some of the layout... I'll test it.

Comment (centipeda) [reply src/components/Settings.vue]: Didn't seem to affect anything... it should work now as a result.

**Comment by stephen-hannon** [comment]
I agree it's not a great user experience to have the image lose its aspect ratio as it reaches small dimensions. I think remembering the original ratio and setting the dimensions based on that would be a good idea.

**Comment by centipeda** [comment]
Implemented. Aspect ratio should be enforced at all times the option is checked, now.

**Review by stephen-hannon — APPROVED** [review]
Looks good! It sometimes slows my computer down when dragging the slider too fast because it tries to compute everything in between, but I'm not exactly sure how to fix that, and I think it's out of the scope of this card.

**Pull request merged**